apps/frontend/src/libs/components/icon/libs/maps/icon-name-to-svg.map.ts

```diff
@@ -1,15 +1,11 @@
 import ArrowRight from "~/assets/img/icons/arrow-right.svg?react";
 import Meetings from "~/assets/img/icons/meetings.svg?react";
 import Settings from "~/assets/img/icons/settings.svg?react";
-import { type IconName } from "~/libs/types/types.js";
 
-const iconNameToSvg: Record<
-	IconName,
-	React.FC<React.SVGProps<SVGSVGElement>>
-> = {
+const iconNameToSvg = {
```

Comment:
Using Record<IconName...> would mandate us adding all icons, which is what we need